radio/src/targets/taranis/board.cpp

```diff
@@ -252,6 +252,9 @@ void boardInit()
   else {
     pwrInit();
     backlightInit();
+#if NUM_PWMANALOGS > 0
+    delay_ms(730);
```

Comment:
wow are you sure?

Reply:
Yup, tested... and kinda makes sense, PWM freq is 240Hz... so with 4 sticks about 1 second is needed to get 1000 cycles! Hence why we would rather want to reduce that number...

Reply:
agreed, I am not sure it's mandatory to wait half a second here just to detect if we have PWM sticks!